tests/test_makefile.mak

```diff
@@ -44,7 +44,7 @@ ifndef XUD_STARTUP_ADDRESS
 $(error XUD_STARTUP_ADDRESS is not set)
 endif
 
-XCC_FLAGS_$(TEST_ARCH)_$(TEST_FREQ)_$(TEST_DTHREADS)_$(TEST_EP_NUM)_$(XUD_STARTUP_ADDRESS) = $(TEST_FLAGS) $(COMMON_FLAGS) 
+XCC_FLAGS_$(TEST_ARCH)_$(TEST_FREQ)_$(TEST_DTHREADS)_$(TEST_EP_NUM)_$(XUD_STARTUP_ADDRESS)_$(BUS_SPEED) = $(TEST_FLAGS) $(COMMON_FLAGS) 
```

Comment:
Where does ```BUS_SPEED``` get passed into the makefile?

Reply:
It will be passed by the build_option here:
for k, v in FIXTURE_TO_DEFINE.items():
        build_options += [str(v) + "=" + str(locals()[k])]